### DISCUSSION TITLE
Revival of Box2D addon

### DISCUSSION THREAD

**Discussion by Tarek-Hasan** [appsinacup/godot-rapier-physics#292] [General]
Hi, I'm just mentioning this, cause you said,

> Currently waiting for [box2c](https://github.com/erincatto/box2c) to be released.

Box2c development has been completed with the release of [Box3D 3.0.0](https://box2d.org/posts/2024/08/releasing-box2d-3.0/) on August 12. Hope this helps.

**Comment by Ughuuu** [comment]
There is this https://github.com/Pizzaandy/godot-box2d-v3/ which continues on it (box2d addon) with version 3. You can follow development on that.
It's wip and there is a discussion around it on Box2D discord https://discord.com/channels/460295137705197568/1294754381903827065
It's not made by me, but by Pizzaandy.

**Comment by Tarek-Hasan** [comment]
Thanks.
